python/ray/autoscaler/commands.py

```diff
@@ -118,8 +119,9 @@ def _bootstrap_config(config):
 
     bootstrap_config, _ = importer()
     resolved_config = bootstrap_config(config)
-    with open(cache_key, "w") as f:
-        f.write(json.dumps(resolved_config))
+    if cache_key:
```

Comment:
This seems like a undefined variable ref.

Reply:
Good catch - looks like I neglected to stage a change declaring ```cache_key = None``` at line 104. 

However, one question I have here is whether we want to write to the local config cache even if --no-config-cache is specified (such that it is primed for any subsequent run without the --no-config-cache flag), or if we want to continue taking the --no-config-cache flag to imply that the config cache is completely disabled for both read and write (which could be safer if a user is experiencing issues writing to the local config cache). 

I lean a bit more toward the latter behavior (disable the config cache for read and write), but don't feel too strongly either way.